packages/ord-connect/src/components/SelectWalletModal/index.tsx

```diff
@@ -396,6 +412,94 @@ export function SelectWalletModal({
     };
   }, [wallet, onConnectUnisatWallet, disconnectWallet]);
 
+  const orderedWalletList = useMemo<WalletListItemProp[]>(() => {
+    const walletList: WalletListItemProp[] = [
+      {
+        wallet: Wallet.OKX,
+        subtitle: "Available on OKX app",
+        onConnect: onConnectOKXWallet,
+        icon: OKXWalletIcon,
+        setErrorMessage,
+        isMobileDevice: isMobile,
+        renderAvatar,
+        isAvailable: !isMobile || (isMobile && network === Network.MAINNET),
+        order: 20,
+      },
+      {
+        wallet: Wallet.UNISAT,
+        subtitle: "Coming soon on mobile browsing",
+        onConnect: onConnectUnisatWallet,
+        icon: UnisatWalletIcon,
+        setErrorMessage,
+        isMobileDevice: isMobile,
+        renderAvatar,
+        isAvailable: !isMobile,
+        order: 21,
+      },
+      {
+        wallet: Wallet.XVERSE,
+        subtitle: "Available on Xverse app",
+        onConnect: onConnectXverseWallet,
+        icon: XverseWalletIcon,
+        setErrorMessage,
+        isMobileDevice: isMobile,
+        renderAvatar,
+        isAvailable: true,
+        order: 22,
+      },
+      {
+        wallet: Wallet.MAGICEDEN,
+        subtitle: "Coming soon on mobile browsing",
+        onConnect: onConnectMagicEdenWallet,
+        icon: MagicEdenWalletIcon,
+        setErrorMessage,
+        isDisabled: isMobile,
+        isMobileDevice: isMobile,
+        renderAvatar,
+        isAvailable: !isMobile,
+        order: 23,
+      },
+      {
+        wallet: Wallet.LEATHER,
+        subtitle: "Coming soon on mobile browsing",
+        onConnect: onConnectLeatherWallet,
+        icon: LeatherWalletIcon,
+        setErrorMessage,
+        isDisabled: isMobile,
+        isMobileDevice: isMobile,
+        renderAvatar,
+        isAvailable: !isMobile,
+        order: 24,
+      },
+    ];
+
+    if (!walletsOrder) {
+      return walletList.sort((a, b) => a.order - b.order);
+    }
+
+    const updatedList = walletList.map((walletItem) => {
+      const foundIndex = walletsOrder.findIndex(
```

Comment:
Why do we need this part?

Reply:
to get new index if the wallet was passed in through the array

Reply:
ahh then I don't think we need to sort it on the line above. Just return walletList directly